platform-ajax/src/main/java/com/softicar/platform/ajax/event/AjaxDomEvent.java

```diff
@@ -53,8 +53,8 @@ public AjaxDomEvent(IDomDocument document, IAjaxRequest ajaxRequest) {
 		this.scrollX = AjaxParameterUtils.getDouble(ajaxRequest, "sx");
 		this.scrollY = AjaxParameterUtils.getDouble(ajaxRequest, "sy");
 		this.keyCode = AjaxParameterUtils.getInteger(ajaxRequest, "k");
-		this.windowWidth = AjaxParameterUtils.getInteger(ajaxRequest, "ww");
-		this.windowHeight = AjaxParameterUtils.getInteger(ajaxRequest, "wh");
+		this.windowWidth = AjaxParameterUtils.getInteger(ajaxRequest, "wx");
```

Comment:
I would have expected a corresponding change on the js/ts side - but there seems to be none. How so?
(same below)

Reply:
there is, it is, see `AjaxRequestMessage.setPoint`